src/xdocs/config_coding.xml

```diff
@@ -7012,21 +7009,21 @@ int x = (a + b) + c;
                 ,<a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#TEXT_BLOCK_LITERAL_BEGIN">
                   TEXT_BLOCK_LITERAL_BEGIN</a>
                 ,<a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LAND">
-                  LAND</a>
+                LAND</a>
```

Comment:
All of these were not fixed. It may be better to revert these back and provide a fix in another PR.